---
title: "Cancer drug resistance causes and categories identified"
date: 2024-10-18
image_filename: "tumour_organoid_WellcomeSangerInstitute.png"
image_alt_text: "A tumoroid, colon cancer cells grown in 3D"
layout: press_release
---
<strong>Researchers have mapped the genetic landscape of cancer drug resistance, uncovering that DNA changes can be grouped into four main categories and highlighting possible new therapeutic targets.</strong>

<br/>

All cancer mutations that cause drug resistance fall into one of four categories. New research has detailed each type, helping to uncover targets for drug development and identify potential effective second-line therapies. 

In a new large-scale study, researchers from the Wellcome Sanger Institute, EMBL’s European Bioinformatics Institute (EMBL-EBI), Open Targets, and collaborators used CRISPR gene editing to map the genetic landscape of drug resistance in cancers, including colon, bone and lung cancer. The team explains how known mutations impact drug resistance and highlights new DNA changes that could be explored further. 

The research, published today (18 October) in Nature Genetics, investigated the effect of mutations on the sensitivity to 10 cancer drugs, also identifying possible effective second-line treatments based on a person’s genetic makeup. 

By understanding the mechanisms of how cancers become resistant to treatment, researchers can identify new targets for personalised therapies, help treat patients based on their cancer's genetic makeup, give second-line treatment options to those who currently have none, and help further research to develop next-generation cancer drugs that could avoid drug resistance emerging. 

One of the major challenges in cancer treatment is drug resistance. Mutations in the cancer cells mean that over time they become less responsive to therapies, and often there are no clear next steps. After cancer has become resistant to the initial treatment, the following therapies are known as second-line therapies and options for these can be limited. Understanding what molecular changes are causing the resistance, and what can be done to tackle this, can help uncover new options and inform clinical paths for specific mutations. 

However, current methods for identifying drug-resistance mutations require multiple samples from patients collected over a long time, making this a time-consuming and difficult process. 

To gather large-scale information on cancer mutations, the team from the Wellcome Sanger Institute, EMBL-EBI, Open Targets, and collaborators used cutting-edge CRISPR gene editing and single-cell genomic techniques to investigate the impact of multiple drugs across human cancer cell lines and organoid cell models. By combining these techniques, researchers were able to create a map showing drug resistance across different cancers focusing on colon, lung, and Ewing sarcoma. The map uncovers more about the mechanisms of drug resistance, highlights DNA changes that may be potential treatment biomarkers, and identifies promising combination or second-line therapies.  

The team found that cancer mutations fall into four different categories depending on the impact of the DNA change. Drug resistance mutations, otherwise known as canonical drug resistance mutations, are genetic changes in the cancer cell that lead to the drug being less effective. For example, changes that mean the drug can no longer bind to its target in the cancer cell. 

Drug addiction mutations lead to some of the cancer cells using the drug to help them grow, instead of destroying them. This research supports the use of drug holidays in the case of drug addiction mutations, which are periods without treatment. This could help destroy the cancer cells with this type of mutation, as the cells are now dependent on the treatment. 

Driver mutations are gain-of-function genetic changes that allow cancer cells to use a different signalling pathway to grow, avoiding the pathway that the drug may have blocked. 

Lastly, drug-sensitising variants are genetic mutations that make the cancer more sensitive to certain treatments and could mean that patients with these genetic variations in their tumour would benefit from particular drugs. 

The research focused on colon cancer, lung cancer, and Ewing sarcoma cancer cell lines as these are all prone to developing resistance and have limited second-line treatments available. The team used 10 cancer drugs that are either currently prescribed or going through clinical trials to help highlight if any of these could be repurposed or used in combination to address resistance, decreasing the time it would take to get any potential treatments to the clinic. 

Understanding more about the four different types of DNA changes can help support clinical decisions, explain why treatments are not working, support the idea of drug holidays in certain patients, and help develop new treatments. This knowledge also helps accelerate drug companies' research into next-generation cancer inhibitors that could better avoid drug resistance.

Dr Matthew Coelho, first author from the Wellcome Sanger Institute and Open Targets, said: “Cancer cells developing resistance to treatments is a huge problem, and having a rapid way to identify these mutations in patients and understand how to combat them is key to treating cancer. Our study details how mutations fall into four different groups, which might need different treatment plans. For example, if there are drug addiction mutations, taking a break from treatment may help. By using cutting-edge genetic techniques, we have started to build a large-scale and rapid way to understand drug resistance and hopefully find new targets for second-line treatments.” 

Dr Magdalena Strauss, study author previously of EMBL’s European Bioinformatics Institute (EMBL-EBI) and now at the University of Exeter, said: “By combining cutting-edge CRISPR gene editing and single-cell techniques with statistical machine learning, we have been able to obtain a detailed picture of the specific mechanisms by which each of the individual mutations that we studied impacts drug response. The functional framework that we have built allows researchers to start to piece together a complete map of common DNA changes seen during cancer treatment, adding to our collective knowledge. It also highlights mutations that could be used as biomarkers, highlighting cancer cells that are more sensitive to certain treatments, which could help inform future clinical trials.”

Dr Mathew Garnett, senior author from the Wellcome Sanger Institute and Open Targets, said: “Before this study, it has been difficult to get a large-scale understanding of why and how drug resistance in cancer develops. This research brings us one step closer to being able to match combination or second-line therapies to a person's genetic make-up, to try and ensure that treatments are as effective and personalised as possible. Additionally, we believe that our new systematic approach will be important for understanding genetic mechanisms of resistance to new drugs in the future. This could help even before the emergence of resistance in the clinic, and these early insights will improve developing cancer treatments.”


<h3>Notes to Editors:</h3>
<strong>Publication</strong>: M.A. Coelho, M.E. Strauss, A. Watterson, et al. (2024) <a href="https://www.nature.com/articles/s41588-024-01948-8">Genetic landscape of cancer drug resistance mechanisms from base editing screens</a>. Nature Genetics. DOI: 10.1038/s41588-024-01948-8 (Open Access)

<strong>Funding</strong>: This research was funded by Wellcome. A full acknowledgement list can be found in the publication. 

<strong>Image credit</strong>: M. Coelho, et al. 2023

<a href="https://www.sanger.ac.uk/news_item/cancer-drug-resistance-causes-and-categories-identified/">This article was originally published by the Wellcome Sanger Institute.</a>
