---
layout: bootstrap
title: Projects
permalink: /projects/
---


Open Targets have identified two connected, over-arching scientific requirements to achieve our vision of systematically identifying and prioritising targets. The first is to integrate information relevant to targets and diseases, and to make it as useful as possible to drug discovery scientists. The [Open Targets Platform](http://targetvalidation.org) has been developed to marshal the resources of existing databases around a common infrastructure to specifically facilitate early target decisions. The second requirement is to increase the wealth of data that provides causal links between targets and diseases. Here, we have developed high throughput experimental projects that generate target-centred data in human, physiologically relevant systems such as organoids and induced pluripotent stem cells. 


<p align="center"><img src="{{ site.url }}/assets/images/knowledge_cycle.png" height="387" width="691"></p>


To deliver our projects Open Targets combines the expertise of its partners in emerging and established technologies such as:
•	Gene editing (CRISPR)
•	Induced pluripotent stem cells
•	Single cell genomics
•	Organoid and tissue culture
•	Large-scale genomics and epigenomics (e.g. ENCODE)
•	Genome wide association studies
•	Next-generation sequencing
•	Bioinformatics
•	High performance computing


# Core Bioinformatics and Data Pipelines
The key challenge for our core bioinformatics work is to integrate data sources that support the validity of a target in a single infrastructure, allowing seamless interrogation of all these data by drug discovery scientists. We aim to cover many data types relevant to human disease biology, and our approach is to leverage the power of existing data analysis and curation efforts within the Open Targets partners and beyond. 


We have developed the [Open Targets Platform](http://www.targetvalidation.org/) to provide summaries of the evidence for the involvement of a specific gene with a selected disease. The platform was developed using a user experience (UX) design process using extensive interaction with a selected user community of scientists within our partners. The first public version of the platform (version 1.0) was released in December 2015. The platform currently supports workflows starting at either a target or disease, and displays the evidence for associations as well as profiles of relevant information for the target or disease and state of the art Javascript visualisations developed and tested via the UX process. We provide direct data access via a supported public API and provide examples via our [blog](https://blog.opentargets.org/).

Our next major bioinformatics focus is the development of a genetics pipeline to establish the most likely variant associations to disease from well-powered GWAS and the association of SNPs to genes via functional genomics data. 

<!-- <iframe src="https://player.vimeo.com/video/149309356" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe> -->

<p><a href="https://vimeo.com/149309356"><img src="{{ site.url }}/assets/images/tvp_-_ot.jpg" alt="Video: Ian Dunham: The Open Targets Platform" /></a></p>


# Collaborative Experimental Projects

<p align="center"><img src="{{ site.url }}/assets/images/Project_grid.png" height="319" width="561"></p>

We have established a portfolio of experimental projects that will provide target identification and prioritisation information relevant to key therapeutic areas. Our approach is to use high throughput methodologies that can address as near as possible the full range of relevant targets (ideally the whole genome) in systems that are relevant to the physiology of diseases with high unmet need.


We currently focus on __Oncology__, __Immunology__ and __Neurodegeneration__ as therapeutic areas for Open Targets both because they represent substantial unmet therapeutic need, and there is complementary expertise within the partners.

## Oncology
<img src="{{ site.url }}/assets/images/oncology.png">

In Oncology there Open Targets has the opportunity to leverage resources and expertise within the Sanger Institute’s cancer program, which has played an important role in understanding the genetic basis of cancer. A shared theme across the oncology workstreams is the application of genomic data from analysis of clinical samples to guide target development. We use a variety of accessible cancer resources to curate and analyse clinical genomic datasets to identify driver genes (mutations, amplification, deletions and gene-fusions) across multiple cancer sub-types. A key resource is the unique collection of >1000 human cancer cell lines at the Sanger Institute along with their drug sensitivities. Genomic information including RNA-seq and synthetic lethality from genome editing will enable the identification of putative targets, the selection of model systems that best reflect the biology of tumours, as well as guide the analysis of experimental results to identify clinically relevant associations.

## Immunology
<img src="{{ site.url }}/assets/images/immunology.png"> 

In Immunology we initially focussed on inflammatory bowel disease (IBD) where there is both a strong interest from GSK as a potential application in the broad immuno-inflammation area and substantial expertise on the Genome Campus. We have developed a state of the art meta-analysis for the existing IBD cohorts, and will move candidate targets from this into genome-wide knockouts in gut epithelium organoids for validation. We are also partly supporting an inception cohort in the UK IBD BioResource, where DNA, biopsies and stool will be collected for future genomic studies. As Open Targets has progressed we have expanded our focus to additional projects probing the role of targets either in well-defined immune cells through gene editing and epigenetic profiling (for instance in macrophages, dendritic cells and T cells in response to various stimulations) or in disease such as asthma using single cell genomics. A project to identify receptor ligand pairs in NK cells with application in immune-oncology crosses these two therapeutic areas.

## Neurodegeneration
<img src="{{ site.url }}/assets/images/neurodegeneration.png"> 

In Neurodegeneration we have used the collective expertise of our partners and also built a collaboration with the Gurdon Institute to generate a set of projects based around Alzheimer’s and Parkinson’s disease. These projects use similar approaches to those described above, such as gene editing in neurons derived from iPS cells to identify modifiers of the response to oxidative stress, mechanism of Tau uptake and the effects of Alzheimer’s disease specific mutations. We are using fine mapping of GWAS in Alzheimer’s and Parkinson’s disease to identify and test potential targets in the same neuron systems and also characterising these systems at the single cell genomics level.
