---
layout: bootstrap
title: Science
permalink: /science/
---


## Scientific Overview
We work in two areas to assist early drug target decisions. First, we integrate publicly available information relevant to targets and diseases in the Open Targets Platform. Second, we perform high throughput experimental projects that generate target-centred data in human, physiologically relevant systems to improve the strength of causal links between targets and diseases in our focus therapeutic areas.


<p align="center"><img src="{{ site.url }}/assets/images/knowledge_cycle.png" height="387" width="691"></p>


To deliver our projects Open Targets combines the expertise of its partners in emerging and established technologies such as:
*	Gene editing (CRISPR)
*	Induced pluripotent stem cells
*	Single cell genomics
*	Organoid and tissue culture
*	Large-scale genomics and epigenomics (e.g. ENCODE)
*	Genome wide association studies
*	Next-generation sequencing
*	Bioinformatics
*	High performance computing


<br><br>

# Core Bioinformatics and Data Pipelines
The big challenge for our core bioinformatics work is how to bring together the various, relevant data on targets in a way that allows simple & seamless exploration by drug discovery scientists. We aim to cover many data types relevant to human disease biology using the expertise from Open Targets partners and beyond. 


We have developed the [Open Targets Platform](http://www.targetvalidation.org/) to provide summaries of the evidence for the involvement of a specific gene with a selected disease. The platform was developed together with scientists from our partners through a user experience (UX) design process. The first public version of the platform (version 1.0) was released in December 2015. The platform supports searches that start at either a target or disease then displays the evidence for associations, as well as profiles of relevant information for the target or disease through state of the art visualisations developed and tested via the UX process. We provide direct data access via a supported public API and provide examples via our [blog](https://blog.opentargets.org/).

Our next major bioinformatics focus is the development of a genetics pipeline to establish the most likely variant associations to disease from well-powered GWAS and the association of SNPs to genes via functional genomics data.  

<!-- <iframe src="https://player.vimeo.com/video/149309356" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe> -->

<p><a href="https://vimeo.com/149309356"><img src="{{ site.url }}/assets/images/tvp_-_ot.jpg" alt="Video: Ian Dunham: The Open Targets Platform" /></a></p>

<br><br>
# Collaborative Experimental Projects
<br>
<p align="center"><img src="{{ site.url }}/assets/images/Project_grid.png" height="319" width="561"></p>
<br><br>

Open Targets portfolio of experimental projects is designed to provide new information in our key therapy areas to enable target identification and prioritisation. We use high throughput methods to address as near as possible the full range of relevant targets (ideally the whole genome) in systems that are most relevant to the disease. We currently focus on __Oncology__, __Immunology__ and __Neurodegeneration__ as therapeutic areas for Open Targets both because they represent substantial unmet patient need, and there is complementary expertise within the partners.

<br>
## Oncology
<p><img src="{{ site.url }}/assets/images/oncology.png"></p> 


Our projects in Oncology are connect to resources and expertise within the Sanger Institute’s cancer program, which has played an important role in understanding the genetic basis of cancer. A shared theme is the use of accessible cancer resources to curate and analyse clinical genomic datasets to identify driver genes (mutations, amplification, deletions and gene-fusions) across multiple cancer sub-types. A key resource is the unique collection of >1000 human cancer cell lines at the Sanger Institute along with their drug sensitivities. Genomic information including RNA-seq and synthetic lethality from genome editing in model systems that best reflect the biology of tumours will identify new target opportunities. 

<br>
## Immunology
<p><img src="{{ site.url }}/assets/images/immunology.png"></p> 


We have initially focussed on inflammatory bowel disease (IBD) where there is both a strong interest from our partners and substantial expertise on the Genome Campus. We have developed a state of the art meta-analysis for the existing IBD cohorts, and will move candidate targets from this into genome-wide knockouts in gut epithelium organoids for validation. We are also partly supporting an inception cohort in the UK IBD BioResource, where DNA, biopsies and stool will be collected for future genomic studies. As Open Targets has progressed we have expanded to additional projects probing the role of targets either in well-defined immune cells through gene editing and epigenetic profiling (for instance in macrophages, dendritic cells and T cells in response to various stimulations) or in disease such as asthma using single cell genomics. A project to identify receptor ligand pairs in NK cells with application in immune-oncology crosses these two therapeutic areas.

<br>
## Neurodegeneration
<p><img src="{{ site.url }}/assets/images/neurodegeneration.png"></p> 

In Neurodegeneration we have used the collective expertise of our partners and through collaboration with the Gurdon Institute built a set of projects based around Alzheimer’s and Parkinson’s disease. These projects use similar approaches to those described above, such as gene editing in neurons derived from iPS cells to identify modifiers of the response to oxidative stress, mechanism of Tau uptake and the effects of Alzheimer’s disease specific mutations. We are using fine mapping of GWAS in Alzheimer’s and Parkinson’s disease to identify and test potential targets in the same neuron systems and also characterising these systems at the single cell genomics level.
<br><br>

# Publications
* [Using human genetics to make new medicines](http://europepmc.org/abstract/MED/26370900) Barrett JC et al.  DOI:10.1038/nrg3998
* [Linking rare and common disease: mapping clinical disease-phenotypes to ontologies in therapeutic target validation](https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0051-7) Sarntivijai S et al. DOI: 10.1186/s13326-016-0051-7
* [Warped Matrix Factorisation](https://link.springer.com/chapter/10.1007%2F978-3-319-46227-1_49) Pratanwanich N et al. DOI:10.1007/978-3-319-46227-1_49
* [TnT: A set of libraries for visualising trees and track-based annotations in the web](https://www.ncbi.nlm.nih.gov/pubmed/27153646) Pignatelli M DOI:10.1093/bioinformatics/btw210
* [Literature Evidence in Open Targets – a target validation platform](https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-017-0131-3)  Kafkas S et al. DOI:10.1186/s13326-017-0131-3
* [Pathway-based dissection of the genomic heterogeneity of cancer hallmarks with SLAPenrich](http://www.biorxiv.org/content/early/2017/03/27/077701) Iorio F et al. DOI:10.1101/077701 
* [Transcription factor activities enhance markers of drug response in cancer](http://www.biorxiv.org/content/early/2017/04/21/129478) Garcia-Alonso L et al. DOI:10.1101/129478
* [Open Targets : a platform for therapeutic target](https://academic.oup.com/nar/article/45/D1/D985/2605745/Open-Targets-a-platform-for-therapeutic-target) Koscielny G et al. DOI:10.1093/nar/gkw1055
* [The ChEMBL Database in 2017](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210557/) Gaulton A et al. DOI:10.1093/nar/gkw1074
