---
layout: default
permalink: /science
title: Our science
---

<header class="py-5" style="background-color:#3489ca">
  <div class="container">
    <div class="row">
      <div class="col-sm-12 text-center text-white">
        <h1>Our science</h1>
      </div>
    </div>
  </div>
</header>

<section class="py-5 my-5">
  <div class="container">
    <div class="row">
      <div class="col-lg-12">
        <p>
          At Open Targets, we are improving how targets are selected for drug discovery. 
        </p>
        <hr class="my-5" />
        <h3>Target selection</h3>
        <p>
          Drugs against genetically validated targets are more likely to progress through clinical trials. The influx of 
          new data from modern genetics and genomics allows targets to be assessed in a comprehensive genome-wide context. 
          In addition, feasibility of new technologies in gene editing, single cell sequencing and cell models allow new 
          types of data to be generated to inform target selection. We bring these approaches together in an integrated 
          pre-competitive research programme. 
        </p> 
        <p>
          We consider target selection to encompass two key steps, target identification and target prioritisation. 
          Target identification defines targets with significant associations to the disease biology. Target prioritisation 
          refines the list of targets on the strength of the association evidence and additional target and disease parameters 
          such as target tractability by drug modality, and likely target-based safety risks. We work together with scientists 
          across our partner organisations to identify areas where our research programme will enhance target identification 
          and prioritisation either by integrating and analysing data, or by generating new data in our therapeutic areas of 
          interest.
        </p>
      </div>
    </div>
    <hr class="my-5" />
    <div class="row">
      <div class="col-lg-12">
        <h3 id="experimental-research-programme" class="mb-3">Generating new data through our Experimental Programme</h3>
        <p>
          We have developed a portfolio of experimental projects designed to provide new information in our key therapy areas 
          to enable target identification and prioritisation. We generate target-centred data in human, physiologically relevant 
          systems to improve the strength of causal links between targets and diseases in our focus therapeutic areas of Oncology, 
          Immunity and Inflammation and Neurodegeneration. We combine whole genome approaches and high throughput methods to 
          address the full range of relevant targets in disease most relevant cellular systems. We use the expertise of all our 
          partners in emerging and established technologies including:
        </p>
        <ul class="mb-5">
          <li>
            Gene editing (CRISPR)
          </li>
          <li>
            Induced pluripotent stem cells
          </li>
          <li>
            Single cell genomics
          </li>
          <li>
            Organoid and tissue culture
          </li>
          <li>
            Large-scale genetics, genomics and epigenomics 
          </li>
          <li>
            Genome-wide sequencing
          </li>
        </ul>
        <div class="d-flex mb-3">
          <img class="align-self-center mr-3" src="/assets/img/icons/oncology-icon.png" style="width:80px">
          <h4 class="d-inline align-self-center">Oncology</h4>
        </div>
        <p class="mb-5">
          Our projects in Oncology utilise resources and expertise within the Wellcome Sanger Institute's 
          <a href="https://www.sanger.ac.uk/science/programmes/cancer-genetics-and-genomics" target="_blank">Cancer, Ageing and Somatic Mutations Program</a>, 
          which has played an important role in understanding the genetic basis of cancer. A shared theme is the use of accessible cancer 
          resources to curate and analyse clinical genomic datasets to identify driver genes (mutations, amplification, deletions 
          and gene fusions) across multiple cancer sub-types. A key resource is the unique collection of >1000 human cancer cell 
          lines at the Sanger Institute along with their drug sensitivities. Genomic information including RNA-seq and synthetic 
          lethality from genome editing in model systems that best reflect the biology of tumours can identify many new target 
          opportunities as exemplified in <a href="https://www.nature.com/articles/s41586-019-1103-9" target="_blank">
          Behan et al 2019</a>. We are expanding our gene editing approaches to both tumour based cancer targets for instance by 
          identifying potential combination targets, and immuno-oncology targets by identifying targets that modify the tumour 
          microenvironment or alter cell killing in co-culture. 
        </p>
        <div class="d-flex mb-3">
          <img class="align-self-center mr-3" src="/assets/img/icons/immunology-icon.png" style="width:80px">
          <h4 class="d-inline align-self-center">Immunology and Inflammation</h4>
        </div>
        <p class="mb-5">
          We focus on explaining the genetic basis of immune-mediated diseases including inflammatory bowel disease (IBD) and SLE where there 
          is a strong interest from our partners and expertise on the Genome Campus. We developed a state of the art meta-analysis for the 
          existing IBD cohorts, and we are moving candidate targets into CRISPR-cas9 knockouts in gut epithelium organoids for validation. 
          In addition, we are generating eQTL datasets including at the single cell level that can be used to resolve the causal genes at 
          loci implicated from genetics for IBD and SLE. We are probing the roles of targets either in well-defined immune cells through 
          gene editing and single cell transcriptomics and epigenetic profiling (for instance in microglia, macrophages, dendritic cells and 
          <a href="https://www.nature.com/articles/s41588-019-0493-9" target="_blank">T cells in response to various stimulations</a>, 
          or in disease such as asthma using <a href="https://www.nature.com/articles/s41591-019-0468-5" target="_blank">single cell genomics</a>. 
          We also have projects that intersect with oncology including identification of receptor ligand pairs in NK cells with application to 
          immuno-oncology as well as intersections with the role of the immune system in neurodegenerative disease.
        </p>
        <div class="d-flex mb-3">
          <img class="align-self-center mr-3" src="/assets/img/icons/neurology-icon.png" style="width:80px">
          <h4 class="d-inline align-self-center">Neurodegeneration</h4>
        </div>
        <p>
          In Neurodegeneration we are using the collective expertise of our partners and through collaboration with UCL and the Cambridge Dementia 
          Research Institute we have built a set of projects based on the use of iPSC derived neurons particularly in Alzheimer's and Parkinson's disease. 
          These projects use CRISPR-cas9 gene editing to identify modifiers of the response to oxidative stress, mechanisms of Tau uptake and the 
          effects of Alzheimer's disease specific mutations in neurons. We also characterise these neurons using single cell transcriptomics.  We have 
          established protocols for similar analysis of enteric and sensory neurons including the effect of rare monogenic mutations 
          for pain. We are using fine mapping of GWAS in Alzheimer’s (AD) and Parkinson’s (PD) disease to identify and test potential targets in the 
          same neuron systems and using functional approaches to model the effect of common variants on relevant phenotypes.
        </p>
        <p>
          The recent discovery of the relevance of neuroinflammation in neurodegeneration has led us to develop projects at the intersection with 
          immunity and inflammation. We have profiled microglia from trauma patients for eQTLs,and are using IPSc-derived microglia for genetic and 
          gene editing screens.
        </p>
        <p>
          Finally, we have recently started to profile the effect of chromatin modifying mutations of neural progenitor cells and neurons, focussing 
          on targets identified in the 
          <a href="https://www.sanger.ac.uk/science/collaboration/deciphering-developmental-disorders-ddd" target="_blank">Deciphering Developmental Disorders project</a>. 
        </p>
      </div>
    </div>
    <hr class="my-5" />
    <div class="row">
      <div class="col-lg-12">
        <h3 id="informatics-research-programme" class="mb-3">Core informatics and data generation pipelines</h3>
        <p class="mb-5">
          Our core bioinformatics work focusses on bringing together the various, relevant data on targets to allow simple and seamless exploration 
          by drug discovery scientists. We cover many data types relevant to human disease biology and target identification using the expertise from 
          Open Targets partners and beyond.
        </p>
        <h4>Open Targets Platform</h4>
        <p class="mb-5">
          Our overall approach to target identification and prioritisation is brought to life in the 
          <a href="https://www.targetvalidation.org" target="_blank">Open Targets Platform</a>, which integrates public data relevant to the association 
          between targets and diseases, and provides additional data and tools for prioritisation. The Platform was developed together with scientists 
          from our partners through a user experience (UX) design process. We integrate data from genetics, somatic mutations, expression analysis, 
          drugs, animal models and the literature through robust pipelines and assess the association of a target with disease in a single score. User 
          friendly interfaces are used to guide the user in target identification and allowing access to the evidence. In addition, prioritisation 
          information is provided by tractability assessment of the target, known safety liabilities, gene expression information and other target 
          properties. The Platform is updated every 2 months in the cloud with separate geolocated instances, is accessible via an application 
          programming interface (API) with <a href="https://docs.targetvalidation.org/programmatic-access/api-tutorials" target="_blank">tutorials</a>, 
          and is developed as an open source project on <a href="https://github.com/opentargets/platform" target="_blank">Github</a> and will continue to 
          evolve as we bring new features and data to bear.
        </p>
        <h4>Open Targets Genetics Portal</h4>
        <p class="mb-5">
          Given the enhanced success of drugs targeting genetically validated targets in clinical trials, we have enhanced the use of human genetics 
          information in target selection. We developed the <a href="https://genetics.opentargets.org/" target="_blank">Open Targets Genetics Portal</a> 
          to provide causal target assignments underlying each association for Genome Wide Association Studies (GWAS). The Portal was initially released 
          at the American Society of Human Genetics conference in October 2018. It supports searches that start with a gene, a single variant, a single 
          study (trait), or multiple studies and users can also 
          <a href="https://genetics-docs.opentargets.org/technical-pipeline/data-download" target="_blank">download the data</a>
          generated for each Portal release. Collaborative work with the <a href="https://www.ebi.ac.uk/gwas/" target="_blank">GWAS catalog</a> 
          allows us to include data from genome-wide summary statistics including UK Biobank. The Portal now also includes workflows to identify 
          candidate causal variants and to analyse colocalization of GWAS and eQTL signals from the new 
          <a href="https://www.ebi.ac.uk/eqtl/" target="_blank">eQTL Catalogue</a> to assist in identifying causal genes. 
          Finally we have developed a machine learning model to assign each association at a locus to the most likely causal gene based on integrated genetics and 
          functional genomics data. The Open Targets Platform receives its common disease genetics information from the Open Targets Genetics Portal, while exporting 
          links for gene and drug information to the Portal.
        </p>
        <h4>Other bioinformatics services and tools</h4>
        <p>          
          Additional ancillary projects in the area of bioinformatics include development of the <a href="https://www.ebi.ac.uk/eqtl/" target="_blank">eQTL Catalogue</a>, 
          network analysis for drug target list expansion, and enhanced data for the Open Targets Platform, including additional data from clinical trial records and data 
          on the effect of mutations on protein function. We also release and support standalone <a href="/resources">informatics tools</a> associated with our various 
          projects.
        </p>
      </div>
    </div>
  </div>
</section>