---
layout: default
permalink: /resources
title: Informatics tools
---

{% include page_header.html %}

<section class="my-5 py-5 ">
  <div class="container">
    <div class="row mb-5">
      <div class="col-lg-12">
        <p>
          We maintain a vareity of specialised, freely-available, open-source informatics tools that help research scientists 
          from around the world answer specific research questions and identify and prioritise potential therapeutic drug targets.
        </p>
      </div>
    </div>

    <div class="row mb-5">

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/SMaSH-framework-screenshot.png" />
          <h4 class="text-center mb-3">
            SMaSH
          </h4>
          <p>
            The <a class="text-center" href="https://gitlab.com/cvejic-group/smash" target="_blank">SMaSH (Scalable Marker gene Signal Hunter)</a>
            framework is a general, scalable codebase for calculating marker genes from single-cell RNA-sequencing
            data for a variety of different cell annotations as provided by the user, using supervised machine learning approaches. For more information, 
            see <a href="https://doi.org/10.1101/2021.04.08.438978" target="_blank">Nelson et al., 2021</a>.  
          </p>
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/covid-19-portal-screenshot.png" />
          <h4 class="text-center mb-3">COVID-19 Target Prioritisation Tool</h4>
          <p>
            The <a class="text-center" href="http://covid19.opentargets.org/" target="_blank">COVID-19 Target Prioritisation Tool</a> is an interactive, 
            open source web portal that integrates molecular and clinical data from EMBL-EBI and other public resources to provide an evidence-based framework to support decision-making on potential drug targets 
            and treatments for <a href="https://platform.opentargets.org/disease/MONDO_0100096" target="_blank">COVID-19</a>.
            For more information, see   
            <a href="https://blog.opentargets.org/2020/07/29/covid-19-target-prioritisation-tool-released/" target="_blank">our release blog post</a>.
          </p>
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/project-score-screenshot.jpg" />
          <h4 class="text-center mb-3">Project Score</h4>
          <p>
            <a class="text-center" href="https://score.depmap.sanger.ac.uk/" target="_blank">Project Score</a> is a web portal that allows 
            researchers to explore the results of CRISPR-Cas9 whole-genome 
            drop out screens across a diverse collection of human cancer cell models and to identify dependencies in 
            cancer cells to help guide early-stage drug discovery for precision cancer medicines. For more information, see  
            <a href="https://www.nature.com/articles/s41586-019-1103-9" target="_blank">Behan et al. 2019</a>.
          </p>
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/eqtl-catalogue-screenshot.jpg" />
          <h4 class="text-center mb-3">eQTL Catalogue</h4>
          <p>
            The <a class="text-center" href="https://www.ebi.ac.uk/eqtl/" target="_blank">eQTL Catalogue</a> provides uniformly processed gene expression and splicing QTLs from all available 
            public studies on human. This resource focuses on expression cis-QTLs where variants are associated 
            to expression levels of nearby genes, and on splicing QTLs where variants are associated to specific 
            splicing events on nearby splice junctions.
          </p>
        </div>
      </div>        

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/cheers-screenshot.jpg" />
          <h4 class="text-center mb-3">CHEERS</h4>
          <p>
            <a class="text-center" href="https://github.com/trynkaLab/CHEERS" target="_blank">CHEERS (Chromatin Element Enrichment Ranking by Specificity)</a>
            is a computational pipeline that determines enrichment of annotations in GWAS significant loci. 
            In addition to SNP-peak overlap, CHEERS takes into account peak properties as reflected by quantitative changes in read counts within peaks. 
            For more information, see <a href="http://dx.doi.org/10.1038/s41588-019-0493-9" target="_blank">Soskic et al. 2019</a>.
          </p>
          <br />
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/effectorness-paper-method-image.jpg" />
          <h4 class="text-center mb-3">Effectorness</h4>
          <p>
            Na√Øve CD4+ T cells coordinate the immune response by acquiring an effector phenotype in response to cytokines. 
            Here, we used quantitative proteomics, bulk RNA-seq, and single-cell RNA-seq  
            to generate a detailed map of cytokine-regulated gene expression programs. For more information, see  
            <a href="https://doi.org/10.1101/753731" target="_blank">Cano Gamez et al. 2019</a> and our
            <a class="text-center" href="/projects/effectorness" target="_blank">Effectorness project page</a>.
          </p>
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/crispr-cleanr-screenshot.png" />
          <h4 class="text-center mb-3">CRISPRcleanR</h4>
          <p>
          CRISPRcleanR is a computational tool that identifies and corrects gene-independent responses to CRISPR-Cas9 
          targeting. CRISPRcleanR is available as an <a href="https://github.com/francescojm/CRISPRcleanR" target="_blank">R package</a>, 
          as a <a href="https://github.com/cancerit/pyCRISPRcleanR" target="_blank">Python package</a>, and
          as a <a href="https://dockstore.org/containers/quay.io/wtsicgp/dockstore-pycrisprcleanr" target="_blank">dockerised, platform-independent Python implementation</a>
          suitable for cloud environments.
          For more information, see see <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4989-y" target="_blank">Iorio et al. 2018</a>.
          </p>
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/cellector-screenshot.jpg" />
          <h4 class="text-center mb-3">CELLector</h4>
          <p>
            <a class="text-center" href="https://ot-cellector.shinyapps.io/CELLector_App/" target="_blank">CELLector</a> is an open source R tool and package that leverages tumour genomics data to explore, rank, and select optimal cell line models, 
            enabling scientists to make appropriate and informed choices about model inclusion and exclusion in retrospective analyses and future studies. For more information, see
            <a href="https://doi.org/10.1016/j.cels.2020.04.007" target="_blank">Najgebauer et al. 2020</a>.
          </p>
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/dorothea-screenshot.jpg" />
          <h4 class="text-center mb-3">DoRothEA</h4>
          <p>
            <a class="text-center" href="http://dorothea.opentargets.io/#/" target="_blank">DoRothEA (Discriminant Regulon Expression Analysis)</a> 
            is a web tool that can be used to search candidate TF-drug interactions in cancer. We studied the role of 127 TFs in drug sensitivity 
            across more than 1,000 cancer cell lines screened with 265 anti-cancer compounds from the <a href="https://www.cancerrxgene.org/" target="_blank">GDSC</a>.
            For more information, see 
            <a href="http://cancerres.aacrjournals.org/content/early/2017/12/09/0008-5472.CAN-17-1679" target="_blank">Garcia-Alonso et al. 2017</a>.
          </p>
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/link-screenshot.png" />
          <h4 class="text-center mb-3">LINK</h4>
          <p>
            <a href="https://blog.opentargets.org/2018/01/18/link/" target="_blank">LINK (LIterature coNcept Knowledgebase)</a> is a now-deprecated pipeline and web tool that allowed for the exploration of 
            half a billion relations between genes, diseases, drugs, and key concepts extracted from PubMed abstracts using 
            natural language processing. With the <a href="https://blog.opentargets.org/next-gen-platform-released/" target="_blank">launch of the next-gen Platform in April 2021</a>, 
            we replaced LINK with a new 
            <a href="https://platform-docs.opentargets.org/bibliography" target="_blank">Bibliography pipeline</a>. 
          </p>
        </div>
      </div>

      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/epichoose-screenshot.jpg" />
          <h4 class="text-center mb-3">epiChoose</h4>
          <p>
            <a class="text-center" href="http://epichoose.shiny.opentargets.io/epiChoose/" target="_blank">epiChoose</a> is a now-deprecated tool for 
            quantifying the relatedness between cell lines and primary cells. 
            We extensively profiled a number of commonly used cell line models across a number of tissues. 
            This profiling consists of epigenetic (histone modification, CTCF, ATAC-seq) and transcriptional 
            (RNA-seq) whole-genome measurements. 
          </p>
        </div>
      </div>


    </div>
  </div>
</section>



